src/sentry/consumers/__init__.py

```diff
@@ -102,6 +102,12 @@ def ingest_monitors_options() -> List[click.Option]:
             default=10,
             help="Maximum time spent batching check-ins to batch before processing in parallel.",
         ),
+        click.Option(
+            ["--max-workers", "max_workers"],
+            type=int,
+            default=None,
```

Comment:
not 100% positive I can set `None` here

Reply:
I think the type only applies to input, but should be easy enough just to test quickly locally

Reply:
Tested this locally. Works as expected yeah.